designs/administrator/4.4/notification-drawer/index.md

```diff
@@ -9,6 +9,11 @@ version: 4.4
 ![](img/notifications-closed.png)
 - When alerts are firing, a special signifier icon appears in the masthead for users with appropriate RBAC permissions. Clicking on the icon opens the notification drawer.
 
+![](img/notifications-icon.png)
+- When there are no notifications, the masthead icon displays only a bell.
+- When there are non-critical notifications (such as updates, lower severity alerts, or other messages) a blue dot appears on the icon along with a number representing the number of items in the drawer,
```

Comment:
Wondering is a number necessary in this case?  If these are informational, it seems like it may not be scalable, and possible distracting if it keeps changing?